lib/game_session.rb

```diff
@@ -1,28 +1,24 @@
 class GameSession
 
-  attr_accessor :player1, :player2, :display1, :rules, :board_size, :board, :current_player
+  attr_accessor :player1, :player2, :display1, :outcome, :board_size, :board, :current_player
 
-  def initialize(player1, player2, display1, rules)
+  def initialize(player1, player2, display1, outcome, initializer)
```

Comment:
Instead of passing the initializer here, we can just pass the board itself.

Reply:
Yes! I've changed it so the board is being passed.